sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_shared/policy.py

```diff
@@ -10,6 +10,7 @@
 import hmac
 from azure.core.pipeline.policies import SansIOHTTPPolicy
 from .utils import get_current_utc_time
```

Comment:
@cochi2 
not sure about this policy change. Can you please review it? Can you please share a related document to this change if any?

Reply:
Synced offline about this. Passing this header is the only way we can pass information from `ContentDownloader` to the Policy, since the policy is not accepting kwargs and the attribute may get lost in the pipeline. 
This header's value is required to correctly sign the download request in a way that AMS is able to validate.